src/com/xilinx/rapidwright/placer/dreamplacefpga/DREAMPlaceFPGA.java

```diff
@@ -186,23 +186,42 @@ public static Design placeDesign(Design design, Path workDir) {
 
         // Load placed result
         Design placedDesign = null;
+        String interchangeResultFile = workDir.toString() + File.separator + "results/design/design";
         try {
-            placedDesign = PhysNetlistReader.readPhysNetlist(interchangeRootFile + Interchange.PHYS_NETLIST_EXT,
+            placedDesign = PhysNetlistReader.readPhysNetlist(interchangeResultFile + Interchange.PHYS_NETLIST_EXT,
                     design.getNetlist());
+            if (makeOutOfContext) {
+                placedDesign.setAutoIOBuffers(false);
+                placedDesign.setDesignOutOfContext(true);
+            }
         } catch (IOException e) {
             throw new UncheckedIOException(e);
         }
         return placedDesign;
     }
 
     public static void main(String[] args) {
-        if (args.length != 2 && args.length != 3) {
-            System.out.println("USAGE: <input DCP> <output DCP> [work_directory]");
+        // Usage: <input DCP> <output DCP> [--out_of_context] [work directory]
+        String MAKE_DCP_OUT_OF_CONTEXT = "--out_of_context";
```

Comment:
You can go ahead and make the variable in `PhysicalNetlistToDcp` `public` so that it can be reused in this context:
https://github.com/Xilinx/RapidWright/blob/a986cf46c4d5667c715a182b653bc516856b2620/src/com/xilinx/rapidwright/interchange/PhysicalNetlistToDcp.java#L39